### DISCUSSION TITLE
How can I use beamnrc simulate Xray tube box? Why the simulation always failed in the middle?

### DISCUSSION THREAD

**Discussion by rrdong** [nrc-cnrc/EGSnrc#1158] [General]
when I tried to simulate the X-ray tube box using tungsten as the outer material, the simulation always failed in the middle every time I put the 0.1cm tungsten at the bottom of the X-ray tube box. Still, when I moved the tungsten at the bottom, it could run successfully, I don't know which the best component module I  should use to simulate the X-ray tube box and why it always stops in the middle of the simulation when I use the flatfilt or conestak as the CM to simulate the x-ray box.
![图片1](https://github.com/nrc-cnrc/EGSnrc/assets/139952747/f0a12cab-8c78-481c-ac3c-20f73ed3c2c7)
![图片2](https://github.com/nrc-cnrc/EGSnrc/assets/139952747/2916fbb6-53ce-4ae8-a72d-15d1b252d302)

**Comment by rtownson** [comment]
Post the .egslst or .egslog file here, so that we can see if there were any other errors?

@blakewalters any ideas?

**Reply by rrdong** [reply]
sure, thank you very much!
These are the .egslst and . errors I got from the failed simulation. I didn't find the .egslog file.
[egsrun_6996_boxtest01_SK-20200423PXTK.zip](https://github.com/user-attachments/files/16168381/egsrun_6996_boxtest01_SK-20200423PXTK.zip)

**Comment by rtownson** [comment]
You can see the message at the bottom of the .egslst file `Rejection plane eliminated more than 1000 fatphotons. STOP`. So it seems there is something not quite right here. The photons should have low weight if they are coming from brems splitting in the target. Double check the DBS and rejection plane set-up, and carefully review the documentation for each (click the "?" boxes in the GUI for some helpful description).

**Comment by blakewalters** [comment]
@rrdong and @rtownson: I think the issue is the layer of lead below the rejection plane where a lot of interactions by phat photons outside the splitting field are happening.  Hence the rejection of > 1000 particles followed by STOP.  What is the output of interest in this simulation?  If it's the phase space just below the air, why bother simulating the bottom lead layer at all?  Or is this intended to be part of a larger simulation where you're interested in quantities downstream of this as well?
